app/interactors/remind_activity_dropoffs.rb

```diff
@@ -0,0 +1,19 @@
+class RemindRegistrationDropoffs
+  include Interactor
+
+  def call
+    context.users = stagnated_users.each do |user|
+      user.update_attributes!(progress_reminder_sent: true)
+      remind(user)
+    end
+  end
+
+  private
+
+  def stagnated_users
+    User.older.with_enrollments.where(progress_reminder_sent_at: nil)
+  end
+
+  def remind(user)
```

Comment:
Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. You can also write as `remind(*)` if you want the method to accept any arguments but don't care about them.